testdata/apply/node1/eth2.nmconnection

```diff
@@ -1,23 +1,11 @@
 [connection]
-id             = eth2
```

Comment:
Since I adde `eth2` to a ovs-bridge, I generated the specific connection files. To avoid conflicting settings I copied over these generated settings, without interpretation of the fields.